update/ansible/playbook/tasks/roles/initialization/tasks/main.yml

```diff
@@ -160,6 +163,12 @@
             - skip_ansible_lint # '503 Tasks that run when changed should likely be handlers'.
       when: is_database_sqlite.stat.exists
 
+    - name: Change default admin id
```

Comment:
I don't think we need it here since the same will run on each container start in `update/ansible/playbook/tasks/update.yml`.

Reply:
it shouldn't, because there are `when` condition for the whole block and `update.yml` doesn't work for docker-way upgrade.